aquadoggo/src/db/stores/test_utils.rs

```diff
@@ -248,7 +248,7 @@ impl TestDatabaseRunner {
             pool.close().await;
 
             // Panic here when test failed to propagate it further
-            assert!(result.is_ok());
+            result.unwrap();
```

Comment:
I believe this will make `#[should_panic]` work again

Reply:
Ah, DOH!!! Yeh, hopefully, why didn't I think of that...

Reply:
I've tested it, it works again :+1: